### PR TITLE
Feature 163 configuration export and import

### PR DESCRIPTION
**Pull request by brodecva**

### CODE REVIEW

File: odalic/pom.xml [hunk 1]
```diff
@@ -39,6 +39,10 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
+		<repository>
+			<id>stardog-public</id>
```
Comment (tomas-knap) [inline odalic/pom.xml]: Just a question, for what purpose do you need stardog libraries?

Comment (brodecva) [inline odalic/pom.xml]: The Pinto library https://github.com/stardog-union/pinto , which generates the RDF model based on class annotations, depends on some of them (e.g. those with groupId com.complexible.common: cp-openrdf-utils and cp-common-utils), as the Pinto is from the same company.

Comment (tomas-knap) [inline odalic/pom.xml]: Btw. Do we know what is the license of pinto?

Comment (brodecva) [inline odalic/pom.xml]: It's Apache 2.

**Pull request merged**